src/main/java/net/minecraftforge/fml/server/FMLServerHandler.java

```diff
@@ -96,6 +100,17 @@ private FMLServerHandler()
     public void beginServerLoading(MinecraftServer minecraftServer)
     {
         server = minecraftServer;
+        ServerEula eula = new ServerEula(new File("eula.txt"));
```

Comment:
If you're going to move this here it might be a good idea to disable the same code in `DedicatedServer`.